pandas/tests/arrays/test_numpy.py

```diff
@@ -211,3 +211,18 @@ def test_basic_binop():
     result = x + x
     expected = PandasArray(np.array([2, 4, 6]))
     tm.assert_extension_array_equal(result, expected)
+
+
+@pytest.mark.parametrize("dtype", [None, object])
```

Comment:
do we have generic setting scalar tests?

Reply:
Yes, TestSetitem does stuff like `x[0] = x[1]`, which we know should work (if it supports setitem).